transformer/python.py

```diff
@@ -686,3 +690,43 @@ def __repr__(self) -> str:
             self.alias,
             self.comments,
         )
+
+
+_T = TypeVar("_T")
+
+
+@dataclass
+class Placeholder(Expression):
+    """
+    The promise of an Expression representing an object currently not in
+    Expression format.
+
+    The Placeholder allows to mix non-Expression objects in the syntax tree,
+    along with a function capable of transforming these objects into actual
+    Expression objects at any time.
+    This is useful when there is a simpler representation than Expression.
+
+    For instance, any Request object can be converted into an equivalent
+    Expression, but Request has a simpler API than Expression for all
+    request-oriented operations like accessing the URL, etc.
```

Comment:
Could you please provide an example of when I would want to use the `Placeholder`?

Reply:
What's more: I wonder if `Placeholder` is only a temporary solution for the time being (until https://github.com/zalando-incubator/Transformer/issues/11 is done) or is it here to stay?

Reply:
I think Placeholder is a useful mechanism that should be part of `transformer.python` in the future.

The paragraph you quoted is supposed to be the example: the `Request` class ends up being converted into Python code (a tree which root is a `python.FunctionCall` object), but it's simpler to use the Request object directly for request-oriented operations like "change the URL". However, Task2 contains both a `request` field and a `statements` field that _has to_ contain the FunctionCall corresponding to that request. To avoid duplicating data, Placeholder is basically a pointer to the `request` field, except it also knows how to "view" `request` as a FunctionCall.

Is it clearer? How would you rephrase the example in Placeholder's documentation?